recipe/meta.yaml

```diff
@@ -13,14 +13,26 @@ source:
 
 build:
   number: 0
-  script: pip install --no-deps .
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
@minrk and @kain88-de  ut seems that we are not ready to use `pip install` on Windows yet :unamused:

With this change everything is passing.

cc @msarahan